FRAuth/FRAuth/Manager/TokenManager.swift

```diff
@@ -365,4 +365,12 @@ struct TokenManager {
         FRUser._staticUser = nil
         Browser.currentBrowser = nil
     }
+    
+    
+    /// Revoke given token
+    func revokeToken(_ token: AccessToken, completion: @escaping CompletionCallback) {
+        self.oAuth2Client.revoke(accessToken: token) { (error) in
```

Comment:
@vahancouver I think this is not going to work.
Calling this will call:
`public func revoke(accessToken: AccessToken, completion: @escaping CompletionCallback) {
        // Construct parameter for the request
        var parameter:[String: String] = [:]
        let token = accessToken.refreshToken ?? accessToken.value
        parameter[OAuth2.token] = token
        parameter[OAuth2.clientId] = self.clientId
        
        let request = Request(url: self.serverConfig.tokenRevokeURL, method: .POST, headers: [:], bodyParams: parameter, urlParams: [:], requestType: .urlEncoded, responseType: .json, timeoutInterval: self.serverConfig.timeout)
        
        FRRestClient.invoke(request: request, action: Action(type: .REVOKE_TOKEN)) { (result) in
            switch result {
            case .success(_ , _):
                completion(nil)
                break
            case .failure(let error):
                completion(error)
                break
            }
        }
    }
    `

Reply:
That code will revoke based on the Refresh token, by default and will thus revoke all accessTokens created by this refresh token. And the refresh token as well. Unless mistaken, this will end up saving a token that has been revoked